charts/argo/templates/server-cluster-role.yaml

```diff
@@ -28,6 +28,9 @@ rules:
   - get
   - list
   - watch
+  - update
+  - patch
+  - delete
```

Comment:
Should we only be adding `delete` here? looking at the manifests over in the Argo repo https://github.com/argoproj/argo/blob/3f8be0cd48963958c493e7669a1d03bb719b375a/manifests/namespace-install/argo-server-rbac/argo-server-role.yaml#L30 looks like it only has `get,list,watch,delete`.

Reply:
If you could also rebase, fix the conflicts and bump the chart version in `Chart.yaml` that would be great :smile:

Reply:
Done!